### ISSUE TITLE
test_streamplot:test_colormap test broke unintentionally

### ISSUE THREAD

**Issue by mdboom** [matplotlib/matplotlib#5549]
In #5307, where I am lowering our testing tolerance down to zero, I am discovering a few recent changes that slipped through because the tolerance on our tests was too high.

This issue is the last of this type that I've found!

`git bisect` points to ba7bc0a71001482e3719afd2ac1dc9ffe165f8d6 as being the first bad commit.

BASELINE:

![streamplot_colormap_test_image-expected](https://cloud.githubusercontent.com/assets/38294/11341875/5071dfa4-91d2-11e5-84a7-5c946330349f.png)

ACTUAL OUTPUT:

![streamplot_colormap_test_image](https://cloud.githubusercontent.com/assets/38294/11341880/56f76966-91d2-11e5-85b9-b4727a997595.png)

TEST CODE:

```
@image_comparison(baseline_images=['streamplot_colormap_test_image'])
def test_colormap():
    X, Y, U, V = velocity_field()
    plt.streamplot(X, Y, U, V, color=U, density=0.6, linewidth=2,
                   cmap=plt.cm.autumn)
    plt.colorbar()
```

Cc: @efiring, Any thoughts?  Replacing the baseline with the new image is a reasonable solution, perhaps, in this case.

**Comment by efiring** [comment]
Yes, I think that replacing the baseline is the thing to do.  The commit you flagged is tightening the criteria for boundary ticks that slightly exceed the boundaries.  It is still able to handle floating point fuzz (if double precision), but in the present instance it is 6 orders of magnitude tighter than the criterion it replaces.  This is another one of these places where there is no perfect solution.  The new algorithm can't be made to exactly duplicate the old results.  It could be loosened so that it would be closer in common cases like this test case.  I wouldn't object to that, but it would be pointless to fiddle with eps until it restores the top tick in this particular test case.

**Comment by mdboom** [comment]
Fixed by #5307.
